src/snowflake/cli/_plugins/nativeapp/entities/application_package.py

```diff
@@ -357,10 +361,22 @@ def action_version_create(
         else:
             git_policy = AllowAlwaysPolicy()
 
-        # Make sure version is not None before proceeding any further.
-        # This will raise an exception if version information is not found. Patch can be None.
         bundle_map = None
-        if not version:
+        version_resolved = None
+        patch_resolved = None
+        label_resolved = ""
+
+        # If version is specified in CLI, no version information from manifest.yml is used (except for comment, we can't control comment as of now).
```

Comment:
did you try if we could set the comment to NULL explicitly? We should file an internal ticket if there's no workaround.

Reply:
the SQL syntaxt does not accept a `comment` parameter. see https://docs.snowflake.com/en/sql-reference/sql/alter-application-package-version#parameters
Will open a ticket internally for it.